__tests__/pet.test.js

```diff
@@ -0,0 +1,187 @@
+const Pet = require('../src/pet');
+const Parent = require('../src/pet');
```

Comment:
this is just the same import twice, but given different names. Should remove the line `const Parent = require('../src/pet');`